docs/topics/i18n/translation.txt

```diff
@@ -418,8 +418,25 @@ Working with lazy translation objects
 -------------------------------------
 
 The result of a ``ugettext_lazy()`` call can be used wherever you would use a
-unicode string (an object with type ``unicode``) in Python. If you try to use
-it where a bytestring (a ``str`` object) is expected, things will not work as
+unicode string (an object with type ``unicode``) in other Django code. It may
+also work with other arbitrary Python code, but there are some exceptions.
+For example::
+
+    # This will not work as expected, because the third-party `requests`
+    # package does not handle Django's ugettext_lazy objects:
+    body = ugettext_lazy(u"I \u2764 Django")  # (unicode :heart:)
```

Comment:
Updating now. This will expand the scope of the change somewhat, as neighboring paragraphs still refer to Python 2 types (and it would seem confusing to leave them mixed).